code/game/objects/items/devices/forcefieldprojector.dm

```diff
@@ -36,6 +37,9 @@
 		return
 	if(get_dist(T,src) > field_distance_limit)
 		return
+	if(get_dist(T,src) < field_distance_min)
```

Comment:
If you're only checking for being on the same tile, you should probably only need to check `if (get_turf(src) == T)`

Reply:
yes